src/components/embed_pages/HolidayEmbedPage.ts

```diff
@@ -0,0 +1,46 @@
+import { EmbedPage } from '../models/EmbedPage';
+import { client } from '../../index';
+
+export const data = new EmbedPage(
+    client,
+    false,
+    false,
+    false,
+    'holiday',
+    'Hoy no hay clase',
+    '',
+    [client.config.generalTextChannelID],
+    [
+        {
+            name: 'Frase insipradora del dia',
+            value: get_alan_kay_phrase(Math.floor(Math.random() * 15)),
+        },
+    ],
+    null,
+    null,
+    client.guilds.cache
+        .first()!
+        .roles.cache.get(client.config.studentRoleID)
+        ?.toString()
+);
+
+function get_alan_kay_phrase(random_number: number): string {
+    let alan_kay_quotes: string[] = [
+        'Most people have managed to get by without being educated…because, in order to make education more user-friendly, they managed to forget about the changes in people´s brains that are supposed to happen.',
+        'Television is the last technology we should be allowed to invent and put out without a surgeon general’s warning.',
+        'In natural science, Nature has given us a world and we’re just to discover its laws. In computers, we can stuff laws into it and create a world.',
```

Comment:
Hay caracteres que no son ascii. No creo que traigan problemas, pero creo que preferiria que si no son nada extravagante, que esten en ascii.

```suggestion
        'Most people have managed to get by without being educated... because, in order to make education more user-friendly, they managed to forget about the changes in people\'s brains that are supposed to happen.',
        'Television is the last technology we should be allowed to invent and put out without a surgeon general\'s warning.',
        'In natural science, Nature has given us a world and we\'re just to discover its laws. In computers, we can stuff laws into it and create a world.',
```